SwiftGen.xcodeproj/project.pbxproj

```diff
@@ -576,6 +579,8 @@
 				09A87B4E1BCCA2C600D9B9F5 /* Resources */,
 				C40C2D80E9C1FEEAA4FD6B14 /* [CP] Embed Pods Frameworks */,
 				6C7EBDA1392A1BFCC9F0A2DA /* [CP] Copy Pods Resources */,
+				1CAEE405320C3DBD251A68B3 /* 📦 Embed Pods Frameworks */,
+				B8FA4CC6E874346645F002E3 /* 📦 Copy Pods Resources */,
```

Comment:
Do you think it would it be beneficial to put a Gemfile in the project to make sure people are on the same version of cocoapods?